analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/Program.cs

```diff
@@ -38,7 +37,12 @@ public static void Main(string[] args)
         {
             if (args.Length == 1)
             {
-                var language = AnalyzerLanguage.Parse(args[0]);
+                var language = args[0] switch
+                {
+                    "cs" => AnalyzerLanguage.CSharp,
+                    "vbnet" => AnalyzerLanguage.VisualBasic,
+                    _ => throw new NotSupportedException("Unsupported argument: " + args[0]),
+                };
```

Comment:
I've moved this functionality out of the class because it has rspec-script specific values, is tooling-only and is not reusable anywhere else.